src/core/ui/markdown/html/DefaultConverter.ts

```diff
@@ -0,0 +1,70 @@
+import { unified } from 'unified';
```

Comment:
I would call this class DefaultConverter or Markdown2htmlConverter or change the file name